### PR TITLE
Added MIGraphX changes

### PR DESCRIPTION
**Pull request by spolifroni-amd**

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Switched to MLIR's 3D Convolution operator.
+* MLIR is now used for Attention operations by default on gfx942 and newer ASICs.
+* Names and locations for VRM specific libraries have changed.
+* Use random mode for benchmarking GEMMs and convolutions.
+* Python version is now printed with an actual version number.
+
+#### Removed
+
+* Disabled requirements for MIOpen and rocBlas when running on Windows.
+* Removed inaccuracte warning messages when using exhaustive-tune.
```
Comment (amd-jnovotny) [inline RELEASE.md:936]:
```suggestion
* Removed inaccurate warning messages when using exhaustive-tune.
```

File: RELEASE.md [hunk 2]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Support for FP8, and INT4
+* Support for the Log2 internal operator
+* Support for the GCC 14 compiler
+* The `BitwiseAnd`, `Scan`, `SoftmaxCrossEntropyLoss`, `GridSample`, and `NegativeLogLikelihoodLoss` ONNX operators 
+* The `MatMulNBits`, `QuantizeLinear`/`DequantizeLinear`, `GroupQueryAttention`, `SkipSimplifiedLayerNormalization`, and `SimpliedLayerNormalizationMicrosoft` Contrib operators 
+* Dynamic batch parameter support to `OneHot` operator
+* Split-K as an optional performance improvement
+* Scripts to validate ONNX models from the ONNX Model Zoo
+* GPU Pooling Kernel
+* `--mlir` flag the migraphx-driver program to offload entire module to mlir
```
Comment (amd-jnovotny) [inline RELEASE.md:914]: MLIR below

File: RELEASE.md [hunk 3]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Fusing split-reduce with MLIR 
+* Multiple outputs for the MLIR + Pointwise fusions 
+* Pointwise fusions with MLIR across reshape operations
+* `MIGRAPHX_MLIR_DUMP` environment variable to dump MLIR modules to MXRs
+* The `3` option to `MIGRAPHX_TRACE_BENCHMARKING` to print the MLIR program for improved debug output
+* `MIGRAPHX_ENABLE_HIPBLASLT_GEMM` environment variable to call hipBlasLt libaries
```
Comment (amd-jnovotny) [inline RELEASE.md:920]:
```suggestion
* `MIGRAPHX_ENABLE_HIPBLASLT_GEMM` environment variable to call hipBLASLt libraries
```

File: RELEASE.md [hunk 4]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+
+* Switched to MLIR's 3D Convolution operator.
+* MLIR is now used for Attention operations by default on gfx942 and newer ASICs.
+* Names and locations for VRM specific libraries have changed.
+* Use random mode for benchmarking GEMMs and convolutions.
+* Python version is now printed with an actual version number.
+
+#### Removed
+
+* Disabled requirements for MIOpen and rocBlas when running on Windows.
```
Comment (amd-jnovotny) [inline RELEASE.md:935]:
```suggestion
* Disabled requirements for MIOpen and rocBLAS when running on Windows.
```

File: RELEASE.md [hunk 5]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
   srcLane, width)` function when one of the parameters to the function is undefined along some path
   to the function. See [issue #3499](https://github.com/ROCm/ROCm/issues/3499) on GitHub.
 
+### **MIGraphX** (2.11.0)
+
+#### Added
+
+* Initial code to run on Windows
+* Support for gfx120x GPU 
+* Support for FP8, and INT4
```
Comment (amd-jnovotny) [inline RELEASE.md:905]:
```suggestion
* Support for `FP8`, and `INT4`
```

File: RELEASE.md (hunk 2 above)
Comment (spolifroni-amd) [reply RELEASE.md]: fixed

File: RELEASE.md [hunk 6]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Removed inaccuracte warning messages when using exhaustive-tune.
+* Remove the hard coded path in `MIGRAPHX_CXX_COMPILER` allowing the compiler to be installed in different locations.
+
+#### Optimized
+
+* Improved:
+    * Infrastructure code to enable better Kernel fusions with all supported data types
+    * Subsequent model compile time by creating a cache for already performant kernels
+    * Use of Attention fusion with models
+    * Performance of the Softmax JIT kernel and of the Pooling opterator
```
Comment (amd-jnovotny) [inline RELEASE.md:945]:
```suggestion
    * Performance of the Softmax JIT kernel and of the Pooling operator
```

File: RELEASE.md [hunk 7]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+    * Tuning operations through a new 50ms delay before running the next kernel
+    * Performance of several convolution based models through an optimized NHWC layout
```
Comment (amd-jnovotny) [inline RELEASE.md:947]:
```suggestion
    * Performance of several convolution-based models through an optimized NHWC layout 
```

File: RELEASE.md [hunk 8]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+#### Optimized
+
+* Improved:
+    * Infrastructure code to enable better Kernel fusions with all supported data types
+    * Subsequent model compile time by creating a cache for already performant kernels
+    * Use of Attention fusion with models
+    * Performance of the Softmax JIT kernel and of the Pooling opterator
+    * Tuning operations through a new 50ms delay before running the next kernel
+    * Performance of several convolution based models through an optimized NHWC layout 
+    * Performance for the FP8 datatype
```
Comment (amd-jnovotny) [inline RELEASE.md:948]:
```suggestion
    * Performance for the `FP8` datatype
```

File: RELEASE.md [hunk 9]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+    * GPU utilization
+    * Verification tools
+    * Debug prints
+    * Documentation, including gpu-driver utility documentation
+    * Summary section of the `migrahx-driver perf` command
```
Comment (amd-jnovotny) [inline RELEASE.md:953]: migrahx or migraphx?

File: RELEASE.md [hunk 10]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Reduced model compilation time
+* Reordered some compiler passes to allow for more fusions
+* Preloaded tiles into LDS to improve performance of pointwise transposes
+* Exposed the `external_data_path` property in `onnx_options` to set the path from `onnxruntime`
+
+#### Resolved issues
+
+* Fixed a bug with gfx1030 that overwrote `dpp_reduce`.
+* Fixed a bug in 1arg dynamic reshape that created a failure.
```
Comment (amd-jnovotny) [inline RELEASE.md:962]: Confirm 1arg, not 1-arg (seen elsewhere)

File: RELEASE.md [hunk 11]
```diff
@@ -897,6 +897,79 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
...
+* Fixed a bug with `dot_broadcast` and `inner_broadcast` that caused compile failures.
+* Fixed a bug where some configs were failing when using exhaustive-tune.
+* Fixed the ROCM Install Guide URL.
```
Comment (amd-jnovotny) [inline RELEASE.md:965]:
```suggestion
* Fixed the ROCm Install Guide URL.
```

File: RELEASE.md [hunk 12]
```diff
@@ -897,6 +897,78 @@ See the full [AMD SMI changelog](https://github.com/ROCm/amdsmi/blob/6.3.x/CHANG
   srcLane, width)` function when one of the parameters to the function is undefined along some path
   to the function. See [issue #3499](https://github.com/ROCm/ROCm/issues/3499) on GitHub.
 
+### **MIGraphX** (2.11.0)
+
+#### Added
+
+* Initial code to run on Windows
+* Support for `FP8`, and `INT4`
```
Comment (amd-jnovotny) [inline RELEASE.md:905]:
```suggestion
* Support for `FP8` and `INT4`
```

**Review by amd-jnovotny — APPROVED** [review]

**Pull request merged**